cabot/urls.py

```diff
@@ -4,35 +4,58 @@
 from django.conf.urls import patterns, include, url
 from django.conf import settings
 from cabot.cabotapp.views import (
-        run_status_check, checks_run_recently, duplicate_check,
-        HttpCheckCreateView, HttpCheckUpdateView,
-        JenkinsCheckCreateView, JenkinsCheckUpdateView,
-        TCPCheckCreateView, TCPCheckUpdateView,
-        StatusCheckDeleteView, StatusCheckListView, StatusCheckDetailView,
-        AuthComplete, LoginError,
-        StatusCheckResultDetailView, StatusCheckReportView,
-        UserProfileUpdateAlert, ScheduleCreateView,
-        ScheduleListView, ScheduleUpdateView, ScheduleDeleteView
+    run_status_check,
+    checks_run_recently,
+    duplicate_check,
+    HttpCheckCreateView,
+    HttpCheckUpdateView,
+    JenkinsCheckCreateView,
+    JenkinsCheckUpdateView,
+    TCPCheckCreateView,
+    TCPCheckUpdateView,
+    StatusCheckDeleteView,
+    StatusCheckListView,
+    StatusCheckDetailView,
+    AuthComplete,
+    LoginError,
+    StatusCheckResultDetailView,
+    StatusCheckReportView,
+    UserProfileUpdateAlert,
+    ScheduleCreateView,
+    ScheduleListView,
+    ScheduleUpdateView,
+    ScheduleDeleteView,
 )
 
 from cabot.cabotapp.views import (
-    ServiceListView, ServiceDetailView,
-    ServiceUpdateView, ServiceCreateView, ServiceDeleteView,
-    UserProfileUpdateView, ShiftListView, subscriptions,
+    ServiceListView,
+    ServiceDetailView,
+    ServiceUpdateView,
+    ServiceCreateView,
+    ServiceDeleteView,
+    UserProfileUpdateView,
+    ShiftListView,
+    subscriptions,
+    ActivityCounterView,
```

Comment:
`ActivityCounterVew`was added.  The rest of the changes to the `import` statements are just cleanup.